setup.cfg

```diff
@@ -1,4 +1,61 @@
 [metadata]
-description_file = README.md
+name = pyfakefs
+version = attr: pyfakefs.__version__
+author = Google
+author_email = google-pyfakefs@google.com
+maintainer = John McGehee
+maintainer_email = pyfakefs@johnnado.com
+license = http://www.apache.org/licenses/LICENSE-2.0
+description = pyfakefs implements a fake file system that mocks the Python file system modules.
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords =
+    testing
+    test
+    file
+    os
+    shutil
+    glob
+    mocking
+    unittest
+    fakes
+    filesystem
+    unit
```

Comment:
probablly remove `glob` (which is no longer mocked) and add `pathlib` instead. Also not sure if `unit` makes sense as a keyword. Ah, and add `pytest`.